src/LEGO.AsyncAPI.Bindings/Sns/Consumer.cs

```diff
@@ -0,0 +1,80 @@
+namespace LEGO.AsyncAPI.Bindings.Sns
+{
+    using System;
+    using System.Collections.Generic;
+    using LEGO.AsyncAPI.Attributes;
+    using LEGO.AsyncAPI.Models.Interfaces;
+    using LEGO.AsyncAPI.Writers;
+
+    public class Consumer : IAsyncApiExtensible
+    {
+        /// <summary>
+        /// What protocol will this endpoint receive messages by?
+        /// </summary>
+        public Protocol Protocol { get; set; }
+
+        /// <summary>
+        /// Where are messages being delivered to?
```

Comment:
it seems odd that these are questions. 😅

Reply:
@VisualBean updated! 🙇‍♂️ (I reflect the change in binding specification as well)